### PR TITLE
add suse sls file for repo.saltstack.com

### PR DESCRIPTION
**Pull request by Ch3LL**

### CODE REVIEW

**Comment by Ch3LL** [comment]
@jfindlay can you please review when you get a sec

File: test_install/saltstack/suse.sls [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{# Import global parameters that source from grains and pillars #}
+{% import 'params.jinja' as params %}
+
+{% set salt_version = salt['pillar.get']('salt_version', '') %}
```
Comment (jfindlay) [inline test_install/saltstack/suse.sls]: @Ch3LL, this should already be defined in `params.jinja`.

File: test_install/saltstack/suse.sls [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+{# Determine Suse_Version #}
+{% set suse_version = params.os + params.os_release %}
```
Comment (jfindlay) [inline test_install/saltstack/suse.sls]: Could you get this with `grains.get('osfullname')`?

**Comment by Ch3LL** [comment]
@jfindlay thanks for the review. I changed suse_version to oscodename, since osfullename did not provide as much information as needed. I also removed salt_version. Once again thank you!

**Pull request merged**